tools/vigiechiro/BilanEnrichiPF.R

```diff
@@ -21,15 +21,15 @@ f2p <- function(x) #get date-time data from recording file names
 
 
 
-IdC2=fread(args[1])
+IdC2=fread(args[1],encoding="UTF-8")
 
 if(substr(IdC2$`nom du fichier`[1],2,2)!="a")
 {
 #  print("Protocole non conforme, ce script doit etre lance uniquement pour un protocole Point Fixe")
   print("Wrong protocol, please only use this tool for a \'Point Fixe\' protocol.")
 }else{
-  refPF=fread(args[2])
-  GroupList=fread(args[3])
+  refPF=fread(args[2],encoding="UTF-8")
+  GroupList=fread(args[3],encoding="Latin-1")
```

Comment:
Why is this Latin, UTF does not work? Mh this is wired.

Reply:
I agree... I think the problem is the original encoding of the file... We are making test on it

Reply:
Re-encoding a file resolved this, we should be able to use UTF-8
Thanks @yvanlebras